Resources/Prototypes/_RMC14/Entities/Structures/Machines/Lathe/lathe_recipes.yml

```
@@ -167,4 +167,11 @@
   result: RMCTablePartReinforced
   completetime: 4
   materials:
-    CMSteel: 400 # 15000
\ No newline at end of file
+    CMSteel: 400 # 15000
+
+- type: latheRecipe
+  id: RMCRackParts
+  result: RMCRackParts
+  completetime: 4
+  materials:
+    CMSteel: 100 # 2500(4687 metal)
```

This should be 125, 4687 / 37.5
```suggestion
    CMSteel: 125 # 4687
```